tests/tween/tween.test.ts

```diff
@@ -4202,5 +4202,39 @@ test('tween custom object, ensure same type', function () {
     expect(target.pos.x).toBe(100);
     expect(target.pos.y).toBe(100);
 
+    director.unregisterSystem(sys);
+});
+
+test('update destroyed node', function () {
+    const sys = new TweenSystem();
+    (TweenSystem.instance as any) = sys;
+    director.registerSystem(TweenSystem.ID, sys, System.Priority.MEDIUM);
+
+    const node = new Node();
+
+    tween(node).to(1, { position: new Vec3(100, 100, 100) }).start();
+
+    runFrames(1);// Start
+
+    runFrames(30);
+    expect(node.position.equals(v3(50, 50, 50))).toBeTruthy();
+
+    node._destroyImmediate();
+
+    expect(node.position).toBeNull();
+    expect(node.scale).toBeNull();
+
+    Tween.stopAllByTarget(node);
```

Comment:
To ensure there are not any script errors while invoking stopAllByTarget. If there are errors, the unit test will fail.